src/ui/ViewModels/FromUcasViewModel.cs

```diff
@@ -36,15 +35,15 @@ public class CourseVariantViewModel
         public string Regions { get; set; }
         public string Status { get; set; }
 
-        public CourseVariantStatus StatusAsEnum => 
+        public CourseVariantStatus StatusAsEnum =>
                 string.Equals("running", Status ?? "", StringComparison.InvariantCultureIgnoreCase) ? CourseVariantStatus.Running
               : string.Equals("not running", Status ?? "", StringComparison.InvariantCultureIgnoreCase) ? CourseVariantStatus.NotRunning
-              : CourseVariantStatus.New;            
-        
+              : CourseVariantStatus.New;
+
         public IEnumerable<SchoolViewModel> Schools { get; set; }
 
     }
-    
+
```

Comment:
We need to think about splitting out this file. It contains 4 separate entities!

Reply:
carded up

https://trello.com/c/lN9Z51QU/235-tidy-up